src/features/profile/ProfileContainer.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+import ProfilePage from './ProfilePage';
+
+const ProfilePageContainer = () => {
```

Comment:
File is called ProfileContainer, but const is ProfilePageContainer

Reply:
Good catch, renamed.